### PR TITLE
Add typos job to GitHub CI devel workflow

### PR DESCRIPTION
**Pull request by kianmeng**
See:
- https://github.com/crate-ci/typos
- https://github.com/joseluisq/static-web-server/pull/140#issuecomment-1249272510

### CODE REVIEW

File: typos.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[default.extend-words]
+Compressable = "Compressable"
+ba = "ba"
+flate = "flate"
+prefered = "prefered"
```
Comment (joseluisq) [inline typos.toml]: Can we move the `typos.toml` to `.github/workflows/config/typos.toml`.

File: .github/workflows/devel.yml [hunk 2]
```diff
@@ -165,6 +165,11 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Check typos
+      uses: crate-ci/typos@master
+      with:
+        config: ./typos.toml
```
Comment (joseluisq) [inline .github/workflows/devel.yml]: After moving `typos.toml` file then of course, we have to update this path.

**Review by joseluisq — CHANGES_REQUESTED** [review]
After the address the feedback we can merge this.

**Review by joseluisq — APPROVED** [review]
Thanks 👍

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**